### PR TITLE
Fix Wildfly docs regarding domain mode

### PR DESCRIPTION
**Pull request by karottenreibe**

### CODE REVIEW

File: agent/README.md [hunk 1]
```diff
@@ -348,12 +348,28 @@ This ensures that the performance of your application does not degrade.
...
 
-Register the agent in the `JAVA_OPTS` environment variable in the `standalone.conf` or `domain.conf`
-file inside the Wildfly installation directory - depending on which "mode" is used; probably standalone.
-
 Please set the agent's `includes` parameter so that the Wildfly code is not being profiled.
 This ensures that the performance of your application does not degrade.
 
+### Using standalone mode
+
+Register the agent in the `JAVA_OPTS` environment variable in the `standalone.conf`
```
Comment (stahlbauer) [inline agent/README.md:356]: Besides `domain.xml` there is a dual `standalone.xml`. Remove the JAVA_OPTS-Approach? Keep it for other/older versions?

File: agent/README.md [hunk 2]
```diff
@@ -348,12 +348,28 @@ This ensures that the performance of your application does not degrade.
...
+file inside the Wildfly installation directory.
+
+### Using domain mode
+
+Register the agent in `domain.xml` under
```
Comment (stahlbauer) [inline agent/README.md:361]: Make the same for `standalone.xml`?

**Review by stahlbauer — CHANGES_REQUESTED** [review]

File: agent/README.md (hunk 1 above)
Comment (karottenreibe) [reply agent/README.md:356]: I would not change the standalone stuff unless we are sure that it needs changing. The last Wildfly I instrumented was in standalone and definitely required changing the conf file

File: agent/README.md (hunk 2 above)
Comment (karottenreibe) [reply agent/README.md:361]: See other comment :-)

**Pull request merged**